api/middleware/apd.js

```diff
@@ -6,25 +6,27 @@ const {
   apdActivity: defaultActivityModel
 } = require('../db').models;
 
-module.exports.loadApd = (
-  model = defaultApdModel,
-  idParam = 'id',
-  ApdModel = defaultApdModel
-) =>
-  cache(['loadApd', modelIndex(model), idParam, ApdModel], () => {
+const apdRelations = (defaultApdModel ? defaultApdModel.withRelated : []).map(
+  relation => `apd.${relation}`
+);
```

Comment:
I'm open to suggestions on better ways of doing this.  The gist of it is when we're loading an APD from a model *other than* the APD model, we want to fetch all of the APD's relationships as well so that we end up with a fully-formed APD object.

Reply:
when would this ternary evaluate to `false`?

Reply:
When the `db.setup()` method is not called first, so the `models` property is an empty object resulting in `const { apd: defaultApdModel } = require('../db').models;` being undefined.

It happens when running the unit tests.  It _shouldn't_ happen otherwise, or else the whole system will crap out, but 🤷‍♂️.  We don't want to run `db.setup()` in the unit tests because then it wants to connect to a real database.  That introduces the risk of forgetting to mock something and then changing the database instead of the mocks.  Shouldn't be a big risk since unit tests are hopefully never running against the production database, but a risk nonetheless.

I'm 100% open to alternatives, I'm not a huge fan of having conditional code that exists solely because of unit tests...